lib/deps-from-ast.js

```diff
@@ -40,19 +40,27 @@ module.exports = function depsFromAST(ast) {
       return;
     }
 
-    if (entry.type === 'CallExpression' && entry.callee.name === 'define') {
-      for (let i = 0; i < entry.arguments.length; i++) {
-        let item = entry.arguments[i];
-        if (item.type === 'ArrayExpression') {
-          for (let j = 0; j < item.elements.length; j++) {
-            let element = item.elements[j];
-            let value = element.value;
-            if (value !== 'exports' && value !== 'require') {
-              imports.push(value);
+    if (entry.type === 'CallExpression') {
+      if (entry.callee.name === 'define') {
+        for (let i = 0; i < entry.arguments.length; i++) {
+          let item = entry.arguments[i];
+          if (item.type === 'ArrayExpression') {
+            for (let j = 0; j < item.elements.length; j++) {
+              let element = item.elements[j];
+              let value = element.value;
+              if (value !== 'exports' && value !== 'require') {
+                imports.push(value);
+              }
             }
+            return STOP;
           }
-          return STOP;
         }
+      } else if (
+        entry.callee.object &&
```

Comment:
Can you replace this with `entry.callee.type === 'MemberExpression' &&`?  It makes checking for the presence of `entry.callee.object` unnecessary, and makes it clearer in the code what you are actually checking for...

Reply:
Nice, I did not like that anyway!